android-23 missing

### Hiya! So you think it's broken or missing a feature eh?

Welllllll, it just might be.  Do me a favor and check the following.

- [x] Issue does not already exist
- Identify exactly what aspect your issue is about
  - [ ] ignite base application
  - [ ] ignite the cli
  - [ ] ignite generated code
  - [x] not sure

Also, for the quickest assistance please run `ignite doctor` and paste in
everything (including the lines with 3 backticks).

I just installed ignite and started the app with `react-native run-android` but I get the below error:
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> failed to find target with hash string 'android-23' in: /usr/local/opt/android-sdk

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

I installed the `android-sdk` with `brew` but I can't find a way to install the version 23, I have only the 24.

```
Computer
  Platform: darwin

Ignite
  Version: 1.10.1
  Path: /Users/<user>/.nvm/versions/node/v6.9.1/bin/ignite

Node
  Version: v6.9.1
  Path: /Users/<user>/.nvm/versions/node/v6.9.1/bin/node

NPM
  Version: 3.10.8
  Path: /Users/<user>/.nvm/versions/node/v6.9.1/bin/npm

Yeoman
  Version: 1.8.5

React Native CLI
  Version: 1.0.0

App
  React Native Version: 0.35.0
```

--- a/ignite-generator/src/app/index.es
+++ b/ignite-generator/src/app/index.es
@@ -177,6 +177,26 @@ export class AppGenerator extends Generators.Base {
   }
 
   /**
+   * Check if Android is good
+   */
+  checkAndroid () {
+    const animation = Utilities.startStep('Finding android', this)
+
+    if (isCommandInstalled('android')) {
+      const androidVersions = Shell.exec('android list', { silent: true }).stdout
+      if (androidVersions.match(/android-23/)) {
+        animation.finish()
+      } else {
+        this.spinner.stop()
+        this.log(`${xmark} Missing android SDK 23`)
+      }
+    } else {
+      this.spinner.stop()
+      this.log(`${xmark} Missing android!`)
+    }
+  }
+
+  /**
    * Check for git.
    */
   findGit () {

